modules/custom/moj_resources/src/SeriesContentApiClass.php

```diff
@@ -101,9 +101,16 @@ public function SeriesNextEpisodeApiEndpoint($lang, $series_id, $number, $episod
   private function decorateSeries($node)
   {
     $results = array_reduce($node, function ($acc, $curr) {
-      $episode_id = ($curr->field_moj_season->value * 1000) + ($curr->field_moj_episode->value);
+      $episode_id = 0;
```

Comment:
We will want to use `camelCase` for variables to be consistent with code styling